float32 front_current
float32 rear_current

